api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -234,45 +234,55 @@
+     * @return the unit of time that the {@link LRA#timeLimit()} element is
      * measured in.
      */
     ChronoUnit timeUnit() default ChronoUnit.SECONDS;
 
     /**
      * <p>
-     * Normally if an LRA is present when a bean method is executed it will not
-     * be ended when the method returns. To override this behaviour and force LRA
-     * termination on exit use the end element.
+     * Normally if an LRA is present when a resource method is executed it will not
+     * be ended when the method returns. To override this behaviour and force the LRA
+     * to close on exit use this <code>end</code> element with the value set to
+     * {@literal true}.
      * </p>
      *
      * <p>
-     * If the <code>end</code> value is set to <code>false</code>
-     * while the annotated method returns the {@link Response} status code
-     * with the meaning the LRA to be cancelled (see {@link #cancelOn()} and {@link #cancelOnFamily()})
-     * then this error state has precedence over the <code>end</code> attribute.
-     * It means the LRA will be cancelled despite the end was set to <code>false</code>.
+     * If the <code>end</code> value is set to {@literal false} but the annotated
+     * method finishes with a status code that matches any of the values specified
+     * in the {@link #cancelOn()} or {@link #cancelOnFamily()} elements
+     * then the  LRA will be cancelled. In other words the
+     * {@link #cancelOn()} and {@link #cancelOnFamily()} elements take precedence
+     * over the <code>end</code> element.
      * </p>
      *
-     * @return true if an LRA that was present before method execution will be
-     * terminated when the bean method finishes.
+     * @return true if an LRA that was active when the method ran should be closed
+     * when the method execution finishes.
      */
     boolean end() default true;
 
     /**
      * <p>
      * The cancelOnFamily element can be set to indicate which families of
-     * HTTP response codes will cause the current LRA to cancel.
-     * This expects the annotated method is the JAX-RS endpoint where the {@link Response}
-     * with the status code could be returned.
+     * HTTP response codes will cause the current LRA to cancel. If the LRA
+     * has already been closed when the annotated method returns then this
+     * element is silently ignored,
      * </p>
      *
      * <p>
-     * By default client errors (<code>4xx codes</code>) and server errors (<code>5xx codes</code>)
+     * If a JAX-RS method is annotated with this element and the method
+     * returns a response code which matches any of the specified families
+     * then the LRA will be cancelled. The method can return status codes
+     * in a {@link Response} or via an exception mappper.
+     * </p>
+     * 
+     * <p>
+     * By default client errors {@link Response.Status.Family#CLIENT_ERROR}
+     * and server errors {@link Response.Status.Family#SERVER_ERROR}
```

Comment:
The **default value** of _cancelOnFamily_ is CLIENT_ERROR and SERVER_ERROR.

I think we should not just  use "By default" as this can cause confusion.

So when the developer set the element value to an empty list, there is no HTTP status anymore which triggers the cancellation of the LRA. So there is no "default behind the scenes" which always kicks in.

Reply:
Okay thanks. I deleted the paragraph and will now merge the PR